MvvmCross-Forms/MvvmCross.Forms.Droid/Platform/MvxFormsAndroidSetup.cs

```diff
@@ -1,28 +1,28 @@
-using System.Collections.Generic;
-using System.Reflection;
 using Android.Content;
 using MvvmCross.Binding;
+using MvvmCross.Binding.Bindings.Target.Construction;
+using MvvmCross.Core.ViewModels;
 using MvvmCross.Droid.Platform;
 using MvvmCross.Droid.Views;
-using MvvmCross.Forms.Bindings;
-using MvvmCross.Forms.Platform;
+using MvvmCross.Forms.Droid.Bindings;
 using MvvmCross.Forms.Droid.Views;
+using MvvmCross.Forms.Platform;
+using MvvmCross.Forms.Views;
 using MvvmCross.Localization;
 using MvvmCross.Platform;
 using MvvmCross.Platform.Droid.Platform;
-using MvvmCross.Core.ViewModels;
 using MvvmCross.Platform.Plugins;
-using MvvmCross.Binding.Bindings.Target.Construction;
-using MvvmCross.Forms.Bindings.Target;
-using MvvmCross.Forms.Views;
-using MvvmCross.Forms.Droid.Bindings;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
 
 namespace MvvmCross.Forms.Droid.Platform
 {
-    public abstract class MvxFormsAndroidSetup : MvxAndroidSetup
+    public abstract class MvxFormsAndroidSetup<TForms> : MvxAndroidSetup
```

Comment:
Why does this need to add a generic? I don't see any value?

Reply:
I'd suggest I revert to non-generic and that we have a separate generic version of the setup class.